tools/clean-filestore-limit.sh

```diff
@@ -23,43 +23,49 @@ if [ -z "$PROJECT_ID" ]; then
 fi
 
 ACTIVE_BUILDS=$(gcloud builds list --project "${PROJECT_ID}" --filter="id!=\"${BUILD_ID}\"" --ongoing 2>/dev/null)
-ACTIVE_FILESTORE=$(gcloud filestore instances list --project "${PROJECT_ID}" --format='value(name)')
-if [[ -z "$ACTIVE_BUILDS" && -z "$ACTIVE_FILESTORE" ]]; then
-	echo "Disabling Filestore API..."
-	gcloud services disable file.googleapis.com --force --project "${PROJECT_ID}"
+if [[ -n "$ACTIVE_BUILDS" ]]; then
+	echo "There are active Cloud Build jobs. Skip clean up."
+	exit 0
```

Comment:
I am wondering if you should not fail the build to clearly signal it did not run.

Reply:
Good point, brining attention may be helpful. Can make it silent in the future if it proves to be noisy/useless.